core/include/jiminy/core/engine/AlgorithmsMotorInertia.h

```diff
@@ -0,0 +1,186 @@
+/// Remimplementation of several dynamics algorithms from pinocchio,
+/// adding support for rotor inertia for 1DoF (prismatic, revolue) joints.
+#ifndef JIMINY_ALGORITHMS_MOTOR_INERTIA_H
```

Comment:
Yes but they are jiminy's own implementation. The documentation is clear about what is being done ; naming in pinocchio would IMHO imply that we are directly calling pinocchio algorithms - like we do in the rest of the code. On the contrary, we are doing our own, completely independant (from a code point for view) implementation.

Reply:
Yes but they overload Pinnochio methods, which are completely dependant of it, and using rather complicated implementation because of it. Why not something like PinocchioOverloadAlgorithmsMotorInertia ?

Reply:
Sure, I get the idea. It's a bit long for a file name though so I simply used PinocchioOverloadAlgorithms